plugins/scaffolder/src/components/JobStage/JobStage.tsx

```diff
@@ -130,8 +136,20 @@ export const JobStage = ({ endedAt, startedAt, name, log, status }: Props) => {
           </Box>
         ) : (
           <Suspense fallback={<LinearProgress />}>
-            <div style={{ height: '20vh', width: '100%' }}>
-              <LazyLog text={log.join('\n')} extraLines={1} follow />
+            <LogModal
+              open={logsFullScreen}
+              onClose={toggleLogsFullScreen}
+              log={log}
+            />
+            <div
+              style={{ height: '20vh', width: '100%' }}
+              onClick={toggleLogsFullScreen}
+            >
+              <LazyLog
+                text={`${log.join('\n')}\nclick to view logs in full screen.`}
```

Comment:
Could you instead make something along the lines of a `position: absolute` div in the upper right corner of the log text that holds a magnifying glass icon button? I would find that a lot clearer

```
,--------------------------------,
| Log Text                ,----, |
| Log Text                | O  | |
|                         |  \ | |
|                         `----` |
|                                |
|                                |
```

Reply:
I opted for more of a MUI style using the `AccordionDetails` with a button to maximize the logs for now.

We can revisit the frontend outside of this PR. :)